auslib/admin/views/scheduled_changes.py

```diff
@@ -74,7 +76,31 @@ def _post(self, form, transaction, changed_by):
         # called "data", this gets overridden, so we need to construct
         # a dict ourselves.
         columns = {k: v.data for k, v in form._fields.iteritems()}
+
+        # The form is checked if its change_type field value is insert
+        # so that it can be initially saved in the rules tables in order to
+        # generate a rule_id before being saved in the schedule change table.
+        # The form also was checked if its an instance of the
+        # ScheduledChangeNewRuleForm class, so as to ensure that only
+        # those form class instances will be affected.
+        if columns["change_type"] == "insert" and isinstance(form, ScheduledChangeNewRuleForm):
```

Comment:
I think this is not required as want rule_id only for update & delete messages(insert dont have a rule_id).
This **might** also be the reason why you are getting the error.